v2.0/enable-node-map.md

```diff
@@ -1,7 +1,200 @@
 ---
-title: Enable the Node Map
+title: Enable the Node Map (Enterprise)
 summary: Learn how to enable the node map in the Admin UI.
 toc: false
 ---
 
-*Docs coming soon*
\ No newline at end of file
+The **Node Map** visualizes the geographical configuration of a multi-regional cluster by plotting the node localities on a world map. The **Node Map** also provides real-time cluster metrics, with the ability to drill down to individual nodes to monitor and troubleshoot the cluster health and performance.
+
+This page walks you through the process of setting up and enabling the **Node Map**.
+
+{{site.data.alerts.callout_info}}The <b>Node Map</b> is an <a href="enterprise-licensing.html">enterprise-only</a> feature. However, you can <a href="https://www.cockroachlabs.com/pricing/request-a-license/">request a trial license</a>  to try it out. {{site.data.alerts.end}}
+
+<div id="toc"></div>
+
+## Setting Up and Enabling the Node Map
```

Comment:
On another look, I'm fine with leaving this as-is, but let's change the heading to `Set Up and Enable the Node Map` to match the form of `Troubleshoot the Node Map`.